controller/contour/controller.go

```diff
@@ -103,26 +103,47 @@ func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 // ensureContour ensures all necessary resources exist for the given contour
 // in namespace ns.
```

Comment:
nit: the godoc comment still refers to `ns`.

Reply:
ack